app/views/hackdays/_project_list.html.erb

```diff
@@ -0,0 +1,36 @@
+<% if hackday.votable %>
```

Comment:
the if and else are almost identical. it would have made more sense to do this later